roles/dhcp/vars/main.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Should we allow users to override these? If so, I'd recommend that we add these to the `defaults` instead as that makes a bit more sense